test/e2e/webhook_test.go

```diff
@@ -0,0 +1,48 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	rukpakv1alpha1 "github.com/operator-framework/rukpak/api/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("bundle api validating webhook", func() {
+	var (
+		bundle *rukpakv1alpha1.Bundle
+		ctx    context.Context
+		err    error
+	)
+	BeforeEach(func() {
+		By("creating the Bundle resource with a long name")
+		ctx = context.Background()
+
+		bundle = &rukpakv1alpha1.Bundle{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "bundlename-0123456789012345678901234567891",
+			},
+			Spec: rukpakv1alpha1.BundleSpec{
+				ProvisionerClassName: plainProvisionerID,
+				Source: rukpakv1alpha1.BundleSource{
+					Type: rukpakv1alpha1.SourceTypeImage,
+					Image: &rukpakv1alpha1.ImageSource{
+						Ref: "testdata/bundles/plain-v0:valid",
+					},
+				},
+			},
+		}
+		err = c.Create(ctx, bundle)
+	})
+	AfterEach(func() {
+		By("deleting the testing Bundle resource for failure case")
+		err := c.Delete(ctx, bundle)
+		Expect(err).NotTo(BeNil())
```

Comment:
We always expect this delete to fail because the bundle wasn't ever created, right? Can we be more explicit about the error we expect?
```suggestion
		Expect(err).To(MatchError(WithTransform(apierrors.IsNotFound), Equal(true))
```

Reply:
Yes, that's right.  I'll update.  Thanks!